src/WorkerChannel.ts

```diff
@@ -61,4 +66,11 @@ export class WorkerChannel {
             after(context);
         }
     }
+
+    public async initAppDir(dir: string) {

```

Comment:
I think this needs to be called from `functionEnvironmentReloadRequest` as well, since that can affect `functionAppDirectory`

Reply:
Done